.github/workflows/ci-unit-tests-go-tip.yml

```diff
@@ -22,11 +24,16 @@ jobs:
     - name: Install Go Tip
       uses: ./.github/actions/setup-go-tip
 
-    - name: Install tools
-      run: make install-test-tools
-
     - name: Run unit tests
       run: make test-ci
 
+    - name: Publish Unit Test Summary 📑
+      uses: EnricoMi/publish-unit-test-result-action@v2
+      id: test-results
```

Comment:
is this needed?

Reply:
Probably not, thanks. Removed in: [5cd1dc0](https://github.com/jaegertracing/jaeger/pull/5030/commits/5cd1dc0720856b28d49f76ef17b8448000fe530b).